apps/prairielearn/src/sync/fromDisk/courseInfo.ts

```diff
@@ -26,7 +26,7 @@ export async function sync(courseData: CourseData, courseId: string) {
     short_name: courseInfo.name,
     title: courseInfo.title,
     display_timezone: courseInfo.timezone || null,
-    example_course: courseInfo.exampleCourse,
+    example_course: undefined, // courseInfo.exampleCourse, // What is this?
```

Comment:
this is not declared on any schemas.

Reply:
Resolved in [18ce57d](https://github.com/PrairieLearn/PrairieLearn/pull/11024/commits/18ce57ddfa550c426105da25509e4fcc88e0fb1a).